config/database.yml

```diff
@@ -17,7 +17,7 @@ default: &default
 
 development:
   <<: *default
-  database: <%= ENV["database"] %>
+  # database: <%= ENV["database"] %>
```

Comment:
[Q] Why do you delete this one. `config/database.yml` should be added to `.gitignore`